src/api/profile.ts

```diff
@@ -2,14 +2,22 @@ import Bugsnag from "@bugsnag/js";
 import firebase from "firebase/app";
 
 import { AnyGridData } from "types/grid";
-import { ProfileLink } from "types/User";
+import { ProfileLink, User } from "types/User";
 import { VenueEvent } from "types/venues";
 
-import { WithVenueId } from "utils/id";
+import { WithId, WithVenueId } from "utils/id";
 
 export const getUserRef = (userId: string) =>
   firebase.firestore().collection("users").doc(userId);
 
+export const getUser = async (userId: string): Promise<WithId<User>> => {
+  const snapshot = await getUserRef(userId).get();
+  return {
+    ...snapshot.data(),
+    id: snapshot.id,
```

Comment:
withId

Reply:
fixed in https://github.com/sparkletown/sparkle/pull/2338/commits/684a3f91796747463ab8f37a3497625fd9d2b89e